playbooks/templates/.env.j2

```diff
@@ -1,118 +1,237 @@
+# routing.
+# For a single server: single-portal.com
+# For a cluster: cluster.com
+PORTAL_DOMAIN={{
+    cluster_domain
+    if portal_cluster_id is defined
+    else server_domain
+}}
+
+{% if portal_cluster_id is defined %}
+# A server domain if a server is a part of the cluster, otherwise keep unset.
+# This is the domain that will be used for generating certificates and nginx
+# routing for cluster server together with PORTAL_DOMAIN.
+# For a single server: keep empty (unset)
+# For a cluster: server-1.cluster.com
+SERVER_DOMAIN={{ server_domain }}
+
+{% endif %}
+# A string representing name of your portal e.g. 'single-portal.com',
+# 'cluster.com' or 'My Awesome Skynet Portal' (internal use only)
+PORTAL_NAME={{ webportal_server_config.portal_name }}
+
+# A list of active portal modules.
+# Order in PORTAL_MODULES defines order of loading modules by docker compose.
+# Possible choices:
+# - 'a': Accounts
+# - 'j': Jaeger
+# Example:
+# PORTAL_MODULES=aj
 PORTAL_MODULES={{ webportal_server_config.portal_modules }}
 
-# absolute url to the server api e.g. https://<subdomain>.siasky.net (direct server
-# address, if this is single server portal use the same address as
-# SKYNET_PORTAL_API)
-SKYNET_SERVER_API={{ webportal_server_config.skynet_server_api }}
+# Absolute url to the server API.
+# For a single server: https://single-portal.com
+# For a cluster: https://server-1.cluster.com
+SKYNET_SERVER_API=https://{{ server_domain }}
 
 # Sia wallet password
-SIA_WALLET_PASSWORD='{{ webportal_server_config.sia_wallet_password if (webportal_server_config.sia_wallet_password is defined and webportal_server_config.sia_wallet_password is not none and webportal_server_config.sia_wallet_password != '') else (webportal_server_config.sia_wallet_seed if webportal_server_config.sia_wallet_seed is defined else '') }}'
+SIA_WALLET_PASSWORD={{ webportal_server_config.sia_wallet_password if (webportal_server_config.sia_wallet_password is defined and webportal_server_config.sia_wallet_password is not none and webportal_server_config.sia_wallet_password != '') else (webportal_server_config.sia_wallet_seed if webportal_server_config.sia_wallet_seed is defined else '') }}
 
-# certificate string that will be used to generate ssl certificates, read more in docker/caddy/Caddyfile
-SSL_CERTIFICATE_STRING={{ webportal_cluster_config.portal_cluster_domain + ', *.' + webportal_cluster_config.portal_cluster_domain + ', *.hns.' + webportal_cluster_config.portal_cluster_domain if (webportal_cluster_config is defined) else webportal_server_config.domain_name }}, *.{{ webportal_server_config.domain_name }}, *.hns.{{ webportal_server_config.domain_name }}
+# Certificate string that will be used to generate ssl certificates, read more in docker/caddy/Caddyfile
+# xxxqqq: Deprecated
+{% if portal_cluster_id is defined %}
+SSL_CERTIFICATE_STRING={{ cluster_domain }}, *.{{ cluster_domain }}, *.hns.{{ cluster_domain }}, *.{{ server_domain }}, *.hns.{{ server_domain }}
```

Comment:
Should we just drop this now?  The update to not need it is being deployed. I

Reply:
Will remove it after 2nd successful deploy. Issue #102